tests/test_src/test_wrappers/test_wrappers.cpp

```diff
@@ -149,3 +150,50 @@ TEST(CryptoWrapper, getSignature)
     EXPECT_EQ(util::stringToHex(signature), expectedSignature);
 
 }
+
+TEST(ClientWrapper, get_validClient)
+{
+    wrapper::http::Client client("https://api.elrond.com");
+    wrapper::http::Result res =  client.get("/address/erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz");
```

Comment:
Maybe use a well-known address (e.g. the staking system smart contract), de-serialize the response into a dummy structure (or dictionary) - yes, the test would be more complex, but more insightful as well - and assert for non-zero balance?

Reply:
I changed the address to : `erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqplllst77y4l`

>de-serialize the response into a dummy structure

I wanted to leave these tests for the ProxyProvider that I'm going to implement in a future ticket. For this particular tests, I just wanted to check that the client wrapper returns a **Result** structure with some valid data.
I can, however, deserialize it and expect more precise/insightful results, it would be better.